code/__DEFINES/bots.dm

```diff
@@ -31,6 +31,7 @@
 #define FLOOR_BOT			4	// Floorbots
 #define CLEAN_BOT			8	// Cleanbots
 #define MED_BOT				16	// Medibots
+#define HONK_BOT            18  //Honkbots
```

Comment:
Should be 32 since these flags represent bits, the one after would be 64 for example

0000 0001 = 1,
0000 0010 = 2,
0000 0100 = 4, doubling the value for each place the 1 moves over